rpcs3/Emu/Cell/lv2/sys_time.cpp

```diff
@@ -252,8 +253,8 @@ error_code sys_time_get_current_time(vm::ptr<s64> sec, vm::ptr<s64> nsec)
 	// Scale nanocseconds
 	tv_nsec = stv_nsec + (tv_nsec * g_cfg.core.clocks_scale / 100);
 
-	// Scale seconds and add from nanoseconds / 1'000'000'000
-	*sec  = stv_sec + (tv_sec * g_cfg.core.clocks_scale / 100u) + (tv_nsec / 1000000000ull);
+	// Scale seconds and add from nanoseconds / 1'000'000'000, and add the console time offset (which might be negative)
+	*sec  = stv_sec + (tv_sec * g_cfg.core.clocks_scale / 100u) + (tv_nsec / 1000000000ull) + g_cfg.sys.console_time_offset;
```

Comment:
```suggestion
	*sec = stv_sec + (tv_sec * g_cfg.core.clocks_scale / 100u) + (tv_nsec / 1000000000ull) + g_cfg.sys.console_time_offset;
```

Reply:
Asking for a reformat of existing code that doesn't align with your preference for spaces is getting a little on the nitpicking side don't you think?

Reply:
deal with it

Reply:
It's normal asking to fix incorrect spacing on a line that you're editing for some other reason, as creating an unique commit to do it will taint the git history unnecessarily